.teamcity/src/subprojects/build/core/ProjectCore.kt

```diff
@@ -53,3 +62,8 @@ object ProjectCore : Project({
         }
     }
 })
+
+fun Requirements.osAndMemoryAgentRequirements(os: String, memoryMb: Int) {
```

Comment:
That's not a very good method name. I thought we just hardcode 7000 here and name the method something like: `requirePowerMachine(os)`.

Reply:
Yes, probably should be prefixed with `define` or `setup`. However I'm not sure how valuable hardcoding the number is in there, especially since `PowerMachine` is relative to what each person understands as power.